raisinbread/instruments/eeg_acquisition_form.rules

```diff
@@ -14,21 +14,29 @@ placement_comment{-}Required when placement is not available{-}placement_rating{
 acquisition_software_other{-}Required when acquisition software is other{-}acquisition_software{@}=={@}other
 recording_reference_other{-}Required when recording reference is other{-}recording_reference{@}=={@}other
 eog_channels_other{-}Required when EOG channels is other{-}eog_channels{@}=={@}other
-artefacted_channels{-}Not Required{-}artefacted_channels{@}=={@}NEVER_REQUIRED
+artefacted_channels{-}Required when artefacted channels are present{-}artefacted_channels_present{@}=={@}yes
 hardware_filters_applied{-}Not Required{-}hardware_filters_applied{@}=={@}NEVER_REQUIRED
 software_filters_applied{-}Not Required{-}software_filters_applied{@}=={@}NEVER_REQUIRED
 resting_state_eeg_incomplete_reason{-}Required when resting state acquiring is incomplete{-}resting_state_eeg_acquired{@}=={@}incomplete
 resting_state_subject_mood_rating_comments{-}Not Required{-}resting_state_subject_mood_rating_comments{@}=={@}NEVER_REQUIRED
 resting_state_comments{-}Not Required{-}resting_state_comments{@}=={@}NEVER_REQUIRED
 task1_eeg_incomplete_reason{-}Required when task 1 acquiring is incomplete{-}task1_eeg_acquired{@}=={@}incomplete
-task1_subject_mood_rating_comments{-}Not Required{-}task1_subject_mood_rating_comments{@}=={@}NEVER_REQUIRED
+task1_attempts{-}Required when task 1 is acquired{-}task1_eeg_acquired{@}=={@}complete|incomplete
+task1_subject_mood_rating{-}Required when task 1 is acquired{-}task1_eeg_acquired{@}=={@}complete|incomplete
+task1_subject_mood_rating_comments{-}Not required{-}task1_subject_mood_rating_comments{@}=={@}NEVER_REQUIRED
 task1_comments{-}Not Required{-}vep_comments{@}=={@}NEVER_REQUIRED
```

Comment:
```suggestion
task1_comments{-}Not Required{-}task1_comments{@}=={@}NEVER_REQUIRED
```
just noticed this field should be `task1` not `vep`.  
- [ ] check and remove: any more instances of `vep`, `aep` and the other old task names? check both the the LINST file and rules

Reply:
Forgot this one. Nice catch.